.meteor/release

```diff
@@ -1 +1 @@
-METEOR@1.4.1.3
+METEOR@1.4.1
```

Comment:
@GathuBoswell @andela-Bgathu Does the deployment still fail without downgrading and/or changing meteor packages?